languages/java/test/TestUtils.java

```diff
@@ -0,0 +1,181 @@
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TestUtils {
+
+    public static final String SAMPLE_SERVICE_RESULT_MESSAGE = "Test Sample Service Result Message";
+
+    public static final String EXPECTED_SERVICE_RESULT_MESSAGE_PATTERN = "(?i)^ServiceResult: \\[id: [a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}, date: [a-z]{3} [a-z]{3} \\d{2} \\d{2}:\\d{2}:\\d{2} [a-z]{3} \\d{4,}, message: %s]$";
```

Comment:
This RegEx pattern needs to be updated to more properly account for valid UUID string representations. The allowed characters are in hexadecimal digits, meaning we should only ever see 0-9 and A-F.

There are technically more constraints/variations in place, especially for different UUID versions and formats, but we don't need to go overboard here. The main idea here was to simply play around with Java's basic RegEx support.


We'll update this properly across utils and tests.